### PR TITLE
Changelog and release

### PR DESCRIPTION
**Pull request by elenavillamil**
Moving release with manage identity and headless services fixes to production.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## [1.0.120221116]
+* Bug fixes for managed identity when using useKubernetesServiceEnvironmentVariables
+* Bug fixes for headless services when using useKubernetesServiceEnvironmentVariables
+
+Thanks @hsubramanianaks and @Tatsinnit for collaborations to make this changes happen.
```
[the remark this answers is not in the document]
Comment (elenavillamil) [reply CHANGELOG.md:8]: Addressed everything.  I didn't add myself since I am the one doing the change so I thought it was implied :)

**Review by Tatsinnit — APPROVED** [review]
🎊❤️☕️ Thanks for adding me into this PR, looks good to me, Thanks heaps.

**Review by hsubramanianaks — APPROVED** [review]
LGTM, Thank you.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Change Log
 
+## [1.0.120221116]
```
Comment (Tatsinnit) [inline CHANGELOG.md:3]: 💡 minor question, why its version: ` [1.0.120221116]` ? and not ` [1.0.120221201]`

**Pull request merged**